src/main/java/io/stargate/sgv2/jsonapi/service/resolver/matcher/FilterResolver.java

```diff
@@ -35,10 +35,6 @@ protected FilterResolver(OperationsConfig operationsConfig) {
 
     matchRules =
         Objects.requireNonNull(buildMatchRules(), "buildMatchRules() must return non null");
-    if (matchRules.isEmpty()) {
```

Comment:
why was this removed ?

Reply:
added back. Though it is unnecessary, since we have matchRules for collection and table.